mungegithub/pulls/ping_ci.go

```
@@ -36,46 +37,29 @@ func init() {
 
 func (PingCIMunger) Name() string { return "ping-ci" }
 
-func (PingCIMunger) MungePullRequest(client *github.Client, pr *github.PullRequest, issue *github.Issue, commits []github.RepositoryCommit, events []github.IssueEvent, opts opts.MungeOptions) {
-	if !HasLabel(issue.Labels, "lgtm") {
+func (PingCIMunger) AddFlags(cmd *cobra.Command) {}
+
+func (PingCIMunger) MungePullRequest(config *config.MungeConfig, pr *github.PullRequest, issue *github.Issue, commits []github.RepositoryCommit, events []github.IssueEvent) {
+	if !github_util.HasLabel(issue.Labels, "lgtm") {
 		return
 	}
-	if pr.Mergeable == nil || !*pr.Mergeable {
+	mergeable, err := config.IsPRMergeable(pr)
+	if !mergeable || err != nil {
```

this is going to eat that error if it's not nil.  at least print it.